Coding-Interview-Codes/Patterns/Pymrid_pattern.cpp

```diff
@@ -0,0 +1,43 @@
+#include<iostream> 
+using namespace std; 
+  

```

Comment:
Change name of file to `Pyramid_pattern.cpp` from `Pymrid_pattern.cpp`

Reply:
Changed name to Pyramid_pattern.cpp